ref/general/deploying-openshift.adoc

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: OpenShift is a Kubernetes-based application platform. After you write your microservice applications, you can containerize and deploy them to OpenShift to orchestrate and automate your containers.
+:seo-description: OpenShift is a Kubernetes-based application platform. After you write your microservice applications, you can containerize and deploy them to OpenShift to orchestrate and automate your containers.
+:page-layout: general-reference
+:page-type: general
+= Deploying Open Liberty applications on OpenShift
+
+link:https://www.openshift.com/[Red Hat OpenShift] is a Kubernetes-based application platform.
+After you write your microservice applications, you can containerize and deploy them to OpenShift to orchestrate and automate your containers.
+
+== What is OpenShift?
+OpenShift is an open source application platform that is based on link:https://kubernetes.io/[Kubernetes].
+Kubernetes is an orchestration platform for containerized applications.
+After you develop and containerize your microservices, you might have many containers that must communicate with each other and scale as each service is needed.
```

Comment:
`you might have many containers that must communicate with each other and scale as each service is needed.`

could be

your containers must be able to communicate with each other and scale as each service is needed.